contrib/action_nitro/Makefile

```diff
@@ -18,3 +21,7 @@ pre-build: src/gen/texts.nit src/gen/planes.nit
 
 check: bin/action_nitro
 	NIT_TESTING=true bin/action_nitro
+
+.PHONY: clean
```

Comment:
* This is generally regarded as a good practice since it clearly states that `clean` is not a file to build.
* This removes one useless check to the filesystem.